### PR TITLE
Release/v2.0.0

### PR DESCRIPTION
**Pull request by simonmitchell**
Release PR for v2.0.0!

### CODE REVIEW

**Comment by simonmitchell** [comment]
No need to look through in super-detail @BenCShutt @RodrigoLondon as this has all been PRd before, but feel free to give it a skim over for any big no-nos!

**Comment by BenCShutt** [comment]
Have we bumped the info.plist?

File: ThunderCloud.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1645,7 +1755,7 @@
 				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
 				LIBRARY_SEARCH_PATHS = "$(inherited)";
-				MARKETING_VERSION = 1.7.7;
+				MARKETING_VERSION = 2.0.0;
```
Comment (simonmitchell) [inline ThunderCloud.xcodeproj/project.pbxproj:1758]: @BenCShutt yep, it's in this hidden file ☺️

Comment (BenCShutt) [reply ThunderCloud.xcodeproj/project.pbxproj:1758]: Oh yes :P

Comment (simonmitchell) [reply ThunderCloud.xcodeproj/project.pbxproj:1758]: (In answer to your question about bumping version)

**Pull request merged**